src/trackinfoobject.h

```diff
@@ -32,10 +32,27 @@ class TrackInfoObject : public QObject {
     Q_OBJECT
 
   public:
-    // Initialize track with a QFileInfo class
-    explicit TrackInfoObject(
+    TrackInfoObject(const TrackInfoObject&) = delete;
+    TrackInfoObject(TrackInfoObject&&) = delete;
```

Comment:
Thank you for the links, Interesting. 

> F.17: 

How do you think about it? 

For me, (coming from plain C) a local object.change() call, the effects an object in the calling function is scary and can be a source of an error. An object->change() is a hint that it is possible. And when we avoid pointer notations in all other cases it becomes a strong hint.  

I looks like there is a back-door for this interpretation: 

> If you prefer the pointer notation (-> and/or * vs. .), not_null<T*> provides the same guarantee as T&.